packages/react-components/src/localization/locales/de-DE/strings.json

```diff
@@ -329,4 +329,4 @@
     "leftNavButtonAriaLabel": "Vorherige Seite",
     "rightNavButtonAriaLabel": "Nächste Seite"
   }
-}
\ No newline at end of file
+}

```

Comment:
are we adding any strings in this change, there are updates in string files.

Reply:
we are not its one of those weird things github does sometimes for no reason. will occasionally for no reason remove the new lines at the end of all the localization files.